docs/release-notes/self-hosted/self-hosted-v9.0.0.md

```diff
@@ -0,0 +1,139 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v9.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.1.25
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.3.0
+---
+
+# Self-hosted v9.0.0
+
+These release notes are for [Codacy Self-hosted v9.0.0](https://github.com/codacy/chart/releases/tag/9.0.0){: target="_blank"}, released on September 21, 2022.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v9.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v9.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the versions with the Git tag `self-hosted-9.0.0`:
+
+    -   [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli/releases/tag/self-hosted-9.0.0)
+    -   [Codacy Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter/releases/tag/self-hosted-9.0.0)
+
+## Breaking changes
+
+This version of Codacy Self-hosted requires [PostgreSQL version 10.20](https://docs.codacy.com/v9.0/chart/requirements/#postgresql-server-setup) since the previous major version [will reach end-of-life (EOL) later this year](https://www.postgresql.org/support/versioning/).
+
+To upgrade PostgreSQL:
+
+1.  Follow the instructions on [how to upgrade a PostgreSQL cluster](https://www.postgresql.org/docs/10/upgrading.html).
+
+1.  After the upgrade, clean the Codacy databases with [vacuumdb](https://www.postgresql.org/docs/10/app-vacuumdb.html) to prevent the <span class="skip-vale">autovacuum</a> feature from momentarily affecting the performance of the databases at a later time.
+
+## Product enhancements
+
+-   The [GitHub integration](https://docs.codacy.com/v9.0/repositories-configure/integrations/github-integration/) now uses a GitHub Apps token instead of a personal one so that the following features continue working even if the person who created the integration leaves the organization on GitHub:
```

Comment:
Not sure how, but maybe we can clarify that this was done for both GitHub Cloud and GitHub Enterprise

Reply:
That's a good idea, @machadoit! I updated the text to make this clear. :+1: